data/tokens.json

```diff
@@ -211,109 +211,197 @@
+          "fontWeight": "{typography.weight.heading}",
+          "lineHeight": "{typography.lineHeight.large}",
+          "fontSize": "{typography.size.h3}",
+          "letterSpacing": "{typography.letterSpacing.base}",
+          "paragraphSpacing": "{typography.paragraphSpacing.base}",
+          "textDecoration": "{typography.textDecoration.base}",
+          "textCase": "{typography.textCase.base}"
         },
         "type": "typography"
       },
-      "H4": {
+      "h4": {
         "value": {
-          "fontFamily": "Mulish",
-          "fontWeight": "700",
-          "lineHeight": "24",
-          "fontSize": "20",
-          "letterSpacing": "0%",
-          "paragraphSpacing": "0",
-          "textDecoration": "none",
-          "textCase": "none"
+          "fontFamily": "{typography.family.base}",
+          "fontWeight": "{typography.weight.heading}",
+          "lineHeight": "{typography.lineHeight.medium}",
+          "fontSize": "{typography.size.h4}",
+          "letterSpacing": "{typography.letterSpacing.base}",
+          "paragraphSpacing": "{typography.paragraphSpacing.base}",
+          "textDecoration": "{typography.textDecoration.base}",
+          "textCase": "{typography.textCase.base}"
         },
         "type": "typography"
       },
-      "H5": {
+      "h5": {
         "value": {
-          "fontFamily": "Mulish",
-          "fontWeight": "700",
-          "lineHeight": "24",
-          "fontSize": "16",
-          "letterSpacing": "0%",
-          "paragraphSpacing": "0",
-          "textDecoration": "none",
-          "textCase": "none"
+          "fontFamily": "{typography.family.base}",
+          "fontWeight": "{typography.weight.heading}",
+          "lineHeight": "{typography.lineHeight.medium}",
+          "fontSize": "{typography.size.body}",
+          "letterSpacing": "{typography.letterSpacing.base}",
+          "paragraphSpacing": "{typography.paragraphSpacing.base}",
+          "textDecoration": "{typography.textDecoration.base}",
+          "textCase": "{typography.textCase.base}"
         },
         "type": "typography"
       },
-      "Body": {
+      "body": {
         "value": {
-          "fontFamily": "Mulish",
-          "fontWeight": "500",
-          "lineHeight": "24",
-          "fontSize": "16",
-          "letterSpacing": "0%",
-          "paragraphSpacing": "0",
-          "textDecoration": "none",
-          "textCase": "none"
+          "fontFamily": "{typography.family.base}",
+          "fontWeight": "{typography.weight.base}",
+          "lineHeight": "{typography.lineHeight.medium}",
+          "fontSize": "{typography.size.body}",
+          "letterSpacing": "{typography.letterSpacing.base}",
+          "paragraphSpacing": "{typography.paragraphSpacing.base}",
+          "textDecoration": "{typography.textDecoration.base}",
+          "textCase": "{typography.textCase.base}"
         },
         "type": "typography"
       },
-      "Footnote": {
+      "footnote": {
```

Comment:
I think I like this lowercase format overall better, feels more consistent with our naming patterns in React